docs/source/user-guide/installation.md

```diff
@@ -28,9 +28,15 @@ Make sure the following requirements are met and software installed on your mach
 
 ## Requirements
 
-- [Apache Spark supported by Comet](overview.md#supported-apache-spark-versions)
-- JDK 8 and up
-- GLIBC 2.17 (Centos 7) and up
+### Supported Apache Spark Versions
+For more details, refer to the documentation: [Apache Spark supported by Comet](overview.md#supported-apache-spark-versions)
+
+- **Apache Spark 3.3, 3.4, or 3.5**
+  - **Supported Java Versions**: JDK 8 and above
+  - **GLIBC**: 2.17 (CentOS 7) and above
```

Comment:
Our latest release is built on Ubuntu 20.04 which has glibc 2.31. Could we update that here?

Reply:
For pre-built binary release, we can mention what glibc we use. Although I think it doesn't mean older glibc is not supported, users still can build by themselves and use older glibc.